Bottom: 2ce87ba65ad7b890efca46ec8b9d56231af9f7fc
Top:    4b508d81e44864a8675a7fdbb0344a6c5e96fa48
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-04-21 02:08:30 +0100

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
